src/main/java/gov/usgs/ngwmn/logic/WaterlevelMediator.java

```diff
@@ -256,7 +256,7 @@ public static BigDecimal mediateToDistanceAboveSiteDatum(
 		unitValidation(measure, siteElevationStr, siteDatumStr);
 		
 		// There is different validation if we are dealing with a measurement that
-		// is from the land surface vs a measurement that is wrt an external datum.
+		// is from the land surface vs a measurement that is based on external datum.
```

Comment:
Thank you!

Reply:
At first, I could not think of what Roger meant my "wrt" so I just reworded the comment.
But now, I realize it is "With Respect To"